ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/BlockValidator.java

```diff
@@ -135,16 +136,20 @@ public SafeFuture<InternalValidationResult> validate(final SignedBeaconBlock blo
                 return reject("Block signature is invalid");
               }
 
+              if (!receivedValidBlockInfoSet.add(new SlotAndProposer(block))) {
+                return ignore("Block has already been imported");
```

Comment:
here I'd say something similar to the first check: "Block is not the first with valid signature for its slot. It will be dropped"